catalog/github/webhook.js

```diff
@@ -1,34 +1,62 @@
 var request = require('request');
 
 /**
- *   Feed to create a webhook on Github
- *  @param {object} params - information about the trigger
- *  @param {string} repository - repository to create webhook
- *  @param {string} username - github username
- *  @param {string} accessToken - github access token
- *  @param {string} events - list of the events the webhook should fire on
- *  @return {object} whisk async
+ * Feed to perform webhook operations on Github
+ *
+ * @param {object}
+ *            triggerName - information about the trigger
+ * @param {string}
+ *            repository - repository to create webhook
+ * @param {string}
+ *            username - github username
+ * @param {string}
+ *            accessToken - github access token
+ * @param {string}
+ *            events - list of the events the webhook should fire on
+ * @param {string}
+ *            operation - the operation the feed should perform, currently
+ *            support list, create, delete
```

Comment:
operation must be lifecycleEvent. It is defined by the feed protocol (I know, not officially documented to satisfaction).